docs/projects/cn0561/index.rst

```diff
@@ -0,0 +1,280 @@
+   * - P13
+     - 1-2
+     - Current sources and fault protection powered from board ±15V sources
+   * - P15
+     - 1-2
+     - Current sources and fault protection powered from board ±15V sources
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL(see more at :ref:`architecture`).
+
+========================  ===========
+Instance                  Address
+========================  ===========
+spi_cn0561_axi_regmap     0x44A0_0000
+axi_cn0561_dma            0x44A3_0000
+odr_generator             0x44B0_0000
+axi_cn0561_clkgen         0x44B1_0000
+========================  ===========
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PS
+     - SPI 0
+     - AD4134
+     - 0
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Software GPIO number is calculated as follows:
+
+-  Zynq-7000: if PS7 is used, then offset is 54
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+   * - cn0561_gpio[7:4]
+     - INOUT
+     - 42:39
+     - 96:93
+   * - cn0561_gpio[2:0]
+     - INOUT
+     - 38:36
+     - 92:90
+   * - cn0561_pinbspi
+     - INOUT
+     - 35
+     - 89
+   * - cn0561_mode
+     - INOUT
+     - 34
+     - 88
+   * - cn0561_pdn
+     - INOUT
+     - 33
+     - 87
+   * - cn0561_resetn
+     - INOUT
+     - 32
+     - 86
+     
+Interrupts
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Below are the Programmable Logic interrupts used in this project.
+
+=============== === ========== ===========
+Instance name   HDL Linux Zynq Actual Zynq
+=============== === ========== ===========
+axi_cn0561_dma  13  57         89
+spi_cn0561      12  56         88
+=============== === ========== ===========
+
+Building the HDL project
+-------------------------------------------------------------------------------
+
+The design is built upon ADI's generic HDL reference design framework.
+ADI does not distribute the bit/elf files of these projects so they
+must be built from the sources available :git-hdl:`here </>`. To get
+the source you must
+`clone <https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository>`__
+the HDL repository, and then build the project as follows:.
+
+**Linux/Cygwin/WSL**
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0561/zed
+   user@analog:~/hdl/projects/cn0561/zed$ make
+
+or
+
+.. code-block::
+   :linenos:
+
+   user@analog:~$ cd hdl/projects/cn0561/coraz7s
```

Comment:
You don't need to give build examples for all the supported carriers. You can keep the first one and delete the others.

Note: We give two examples only if there are build parameters that can be set during make.